README.md

```diff
@@ -1 +1,67 @@
-# diablo
\ No newline at end of file
+# Diablo
+
+UC Berkeley at Course Capture
```

Comment:
This typo will be fixed in the next PR.

Reply:
UCC Berkeley at California. UBCalifornia at Capture. UniCourseBCaptured at USB-C.

Reply:
This is a Diablo comment.